coldfront/plugins/qumulo/views/update_allocation_view.py

```diff
@@ -61,7 +77,55 @@ def get_form_kwargs(self):
         kwargs["initial"] = form_data
         return kwargs
 
-    def form_valid(self, form: UpdateAllocationForm):
+    def _acl_reset_message(self):
+        name  = Allocation \
+                    .objects \
+                    .get(pk=self.kwargs.get("allocation_id")) \
+                    .get_attribute(name='storage_name')
+        return (
+            f'ACL reset initiated for {name}.  '
+            'An e-mail notification will be sent when the task completes.'
+        )
+
+    def _allocation_linkage_exists(self):
+        has_linkage = True
+        try:
+            AllocationLinkage.objects.get(
+                parent=Allocation.objects.get(
+                    pk=self.kwargs.get("allocation_id")
+                )
+            )
+        except AllocationLinkage.DoesNotExist:
+            has_linkage = False
+        return has_linkage
+
+
+    def _reset_acls(self):
+        # bmulligan (20240903): "retry" and "timeout" are intended to be
+        # arbitrarily high values.  It would be good if the application could
+        # know or learn what they should be.  Testing has shown that the DEV
+        # infrastructure can process a directory tree of about 90,500 items in
```

Comment:
RE: knowing how long - SysEng is setting up DataDog.  That *should* (if incorporated with the app) be able to show us a graph of completion times for tasks by type.

Meanwhile, we could just log start/end times inside the tasks running on qcluster.

Reply:
It looks like start and end times ("started" and "stopped" in the admin console under successful tasks) are logged by the application.